plugins/keyboard-navigation/test/navigation_modify_test.mocha.js

```diff
@@ -0,0 +1,476 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const chai = require('chai');
+const Blockly = require('blockly/node');
+const {Navigation} = require('../src/navigation');
+const assert = chai.assert;
+
+suite('Insert/Modify', function() {
+  /**
+   * Captures the strings sent to console.warn() when calling a function.
+   * Copies from core.
+   * @param {function} innerFunc The function where warnings may called.
+   * @return {string[]} The warning messages (only the first arguments).
+   */
+  function captureWarnings(innerFunc) {
+    const msgs = [];
+    const nativeConsoleWarn = console.warn;
+    try {
+      console.warn = function(msg) {
+        msgs.push(msg);
+      };
+      innerFunc();
+    } finally {
+      console.warn = nativeConsoleWarn;
+    }
+    return msgs;
+  }
```

Comment:
I think we should consider providing this helper in dev-tools plugin. Maybe not in this PR, but file a bug.

Reply:
done. #567